.importlinter

```diff
@@ -41,4 +41,5 @@ name = Core App Dependency Layering
 type = layers
 layers=
     openedx_learning.core.components
+    openedx_learning.core.contents
     openedx_learning.core.publishing
```

Comment:
I love this--it conveys so much about the relationship between the apps in one glance.